specs/deneb/p2p-interface.md

```diff
@@ -184,6 +185,7 @@ The following validations MUST pass before forwarding the `blob_sidecar` on the
 - _[REJECT]_ The sidecar's inclusion proof is valid as verified by `verify_blob_sidecar_inclusion_proof(blob_sidecar)`.
 - _[REJECT]_ The sidecar's blob is valid as verified by `verify_blob_kzg_proof(blob_sidecar.blob, blob_sidecar.kzg_commitment, blob_sidecar.kzg_proof)`.
 - _[IGNORE]_ The sidecar is the first sidecar for the tuple `(block_header.slot, block_header.proposer_index, blob_sidecar.index)` with valid header signature, sidecar inclusion proof, and kzg proof.
+*Note:* If client obtains the blob via local execution layer client or any other method before receiving the corresponding sidecar via this topic, it MUST NOT consider the tuple as "seen" (i.e. it MUST participate in the dissemination of the blob_sidecar even if it already obtained the blob)
```

Comment:
@mkalinin not sure about this specification now that there is a requirement to publish the `blob_sidecar` whenever is recovered.
WDYT?

Reply:
I think it makes sense to mention how to deal with the sidecar gossip in the EL blob recovery section. i.e. mention that if the sidecar has been published after EL blob recovery then it must also be considered as seen.

Reply:
I updated it